npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -136,19 +136,24 @@ export async function generateTsConfig (devServerConfig: AngularWebpackDevServer
     includePaths.push(...polyfills.map((p: string) => getProjectFilePath(workspaceRoot, p)))
   }
 
-  const cypressTypes = getProjectFilePath(workspaceRoot, 'node_modules', 'cypress', 'types', 'index.d.ts')
+  const typeRoots = [
+    getProjectFilePath(workspaceRoot, 'node_modules'),
+    getProjectFilePath(workspaceRoot, 'node_modules', 'types'),
+  ]
 
-  includePaths.push(cypressTypes)
+  const types = ['cypress']
 
   const tsConfigContent = JSON.stringify({
     extends: getProjectFilePath(projectRoot, buildOptions.tsConfig ?? 'tsconfig.json'),
     compilerOptions: {
       outDir: getProjectFilePath(projectRoot, 'out-tsc/cy'),
       allowSyntheticDefaultImports: true,
       skipLibCheck: true,
+      types,
+      typeRoots,
     },
     include: includePaths,
-  })
+  }, null, 2)
```

Comment:
I'm assuming this was just to get pretty formatting? Still needed since this is dumping to a temp dir?

Reply:
Not needed, but makes it "pretty"er 🤣